src/components/dialog/Dialog.js

```diff
@@ -45,16 +45,24 @@ class Dialog extends PureComponent {
     ]);
 
     return (
-      <DialogBase className={classNames} {...restProps}>
+      <DialogBase className={classNames} {...restProps} scrollable={false}>
         {title && this.getHeader()}
-        {children}
+        {scrollable ? (
+          <Box display="flex" flexDirection="column" overflowY="auto">
```

Comment:
Q: Any specific reason for `flex` here? This can just be block, right?

Reply:
A: I need to set the same flex props like the `inner` container currently has. Otherwise, there will be the need to add a `class` or `ref` to the new scrollable container later on.